Modules/Core/Services/ReportsService.php

```
@@ -0,0 +1,130 @@
+     * @param int|null $minQuantity Minimum quantity filter
+     * @param int|null $maxQuantity Maximum quantity filter
+     * @param bool|null $checkboxTax Include tax in calculations
+     *
+     * @return array Report results
+     *
+     * @legacy-function salesByYear
+     */
+    public function salesByYear(
+        string $fromDate,
+        string $toDate,
+        ?int $minQuantity = null,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$minQuantity'. (undefined)

(UnusedFormalParameter)